### PR TITLE
kisuk_playground: Data Pipeline Changes

### PR DESCRIPTION
**Pull request by supersergiy**

### CODE REVIEW

File: zetta_utils/layer/volumetric/tools.py [hunk 1]
```diff
@@ -121,6 +133,53 @@ def process_data(self, data: torch.Tensor, mode: Literal["read", "write"]) -> to
         return result
 
 
+@builder.register("ROIMaskProcessor")
+@typechecked
+@attrs.mutable
+class ROIMaskProcessor(JointIndexDataProcessor):  # pragma: no cover
```
Comment (dodamih) [inline zetta_utils/layer/volumetric/tools.py:139]: This needs a comment since the `_mask` naming convention is hardcoded.

**Review by dodamih — CHANGES_REQUESTED** [review]

File: zetta_utils/layer/volumetric/tools.py (hunk 1 above)
Comment (supersergiy) [reply zetta_utils/layer/volumetric/tools.py:139]: Good catch. @torms3 could you add a comment explaining how to use this guy?
I'll write a test for `process_data` in the meanwhile, it's too complicated for `no cover`

Comment (supersergiy) [reply zetta_utils/layer/volumetric/tools.py]: Done with the test: https://github.com/ZettaAI/zetta_utils/pull/584/commits/97fe120e113a66b3784d6ada5f8a84525f388d31

Comment (torms3) [reply zetta_utils/layer/volumetric/tools.py]: I added a docstring: [``2ed8dad``](https://github.com/ZettaAI/zetta_utils/pull/584/commits/2ed8dad5a6e0cae36cba9e406ce0097d4fcf399f)

How does it sound? @dodamih @supersergiy

Comment (supersergiy) [reply zetta_utils/layer/volumetric/tools.py]: Thanks, looks great!

**Pull request merged**